src/app/components/Dialog/Portal.tsx

```diff
@@ -1,14 +1,15 @@
-import React, { useMemo } from 'react';
+import React, { useMemo, useId } from 'react';
 import reactDom from 'react-dom';
 
 function Portal(props: { children: React.ReactNode }) {
   const { children } = props;
+  const elementId = useId();
 
   const newElement = document.createElement('div');
-  newElement.id = 'portal-root';
+  newElement.id = 'portal-root-' + elementId;
```

Comment:
```suggestion
  newElement.id = `portal-root-${elementId}`;
```

이렇게하면 읽는입장에서 조금 더 좋을 것 같습니다 :)

Reply:
넹!